test_files/enum.sickle.ts

```diff
@@ -13,4 +13,19 @@ export enum EnumTest2 {XYZ, PI = 3.14159}
 (<any>EnumTest2).XYZ = 0;
 /** @type {EnumTest2} */
 (<any>EnumTest2).PI =  3.14159;
+/** @typedef {number} */
```

Comment:
Change the above call from `this.writeNode(node);` to `this.writeTextBetween(node.getStart(), node.getEnd())` to try working around this.